core/src/test/java/io/parsingdata/metal/BackTrackNameBindTest.java

```diff
@@ -42,10 +46,10 @@
 
     private final Token _choiceRef = seq(any("a"),
                                          cho(seq(any("a"), eqRef("b", "a")),
-                                             seq(notEqRef("b", "a"), any("c"))));
+                                             seq(def("b", con(1), not(Shorthand.eq(ref("a")))), any("c"))));
```

Comment:
Static import `eq`.

Reply:
It maybe clashes with another imported ```eq```.

Reply:
Ah, missed that one. 👍